egs/lre07/v2/run.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+# Copyright  2016   GoVivace Inc. (Author: Mousmita Sarma) 
+#                   
+# Apache 2.0.
+#
+# This script runs the NIST 2007 General Language Recognition Closed-Set
+# evaluation.
+# This example script shows how to replace the GMM-UBM
+# with a DNN trained for ASR. 
+
+. cmd.sh
+. path.sh
+set -e
+
+mfccdir=`pwd`/mfcc
+vaddir=`pwd`/mfcc
+languages=local/general_lr_closed_set_langs.txt
+nnet=exp/nnet2_online/nnet_ms_a/final.mdl
+
+# Train a DNN on about 1800 hours of the english portion of Fisher.
+local/dnn/train_dnn.sh
+
+data_root=/export/corpora/LDC
+# Training data sources
+local/make_sre_2008_train.pl $data_root/LDC2011S05 data
+local/make_callfriend.pl $data_root/LDC96S60 vietnamese data
+local/make_callfriend.pl $data_root/LDC96S59 tamil data
+local/make_callfriend.pl $data_root/LDC96S53 japanese data
+local/make_callfriend.pl $data_root/LDC96S52 hindi data
+local/make_callfriend.pl $data_root/LDC96S51 german data
+local/make_callfriend.pl $data_root/LDC96S50 farsi data
+local/make_callfriend.pl $data_root/LDC96S48 french data
+local/make_callfriend.pl $data_root/LDC96S49 arabic.standard data
+local/make_callfriend.pl $data_root/LDC96S54 korean data
+local/make_callfriend.pl $data_root/LDC96S55 chinese.mandarin.mainland data
+local/make_callfriend.pl $data_root/LDC96S56 chinese.mandarin.taiwan data
+local/make_callfriend.pl $data_root/LDC96S57 spanish.caribbean data
+local/make_callfriend.pl $data_root/LDC96S58 spanish.noncaribbean data
+local/make_lre96.pl /export/corpora/NIST/lid96e1 data
```

Comment:
You can remove line 39, since all the data in lid96e1 is included in LDC2006S31.